### DISCUSSION TITLE
Quickly muting an alert

### DISCUSSION THREAD

**Discussion by jayluo667** [ccfos/nightingale#1304] [Q&A] [answered]
情景：接下来已准备维护某服务，不想通过屏蔽规则去操作（时长限定，不想手动），某服务（维护时间每周不固定，维护时长不固定）
问题：有相应的API接口或其他方式可解决这类场景吗？求大佬指点

**Comment by UlricQin** [comment, marked as answer]
可以，页面上的所有功能都可以通过api实现：http://n9e.flashcat.cloud/docs/api/webapi/ 相当于你用程序伪装成一个人去登录操作

**Reply by jayluo667** [reply]
> 可以，页面上的所有功能都可以通过api实现：http://n9e.flashcat.cloud/docs/api/webapi/ 相当于你用程序伪装成一个人去登录操作

大佬，想请教下，比方说想启用或者关闭某个屏蔽规则，找到接口http://xxx.com/api/n9e/busi-group/2/alert-mutes，我该如何知道哪些参数是必填项，对应的启用或关闭操作对应的是参数名是啥（用惯那些API接口文档）？我只是个小菜鸟运维，不太懂，烦请大佬不吝指教

**Comment by UlricQin** [comment]
你在页面上直接操作相关功能，哪些字段是必填项肯定会提示你的。打开chrome开发者工具，看看发的http request，照着来就行了
